src/test/java/com/training/services/tests/ProductsTest.java

```diff
@@ -0,0 +1,48 @@
+package com.training.services.tests;
+
+import com.training.base.BaseTest;
+import com.training.utilities.RestEngine;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ProductsTest extends BaseTest {
+    Map<String, Object> headerMap;
+    Map<String, Object> queryParam;
+    String baseURL;
+    com.training.services.products.Response response;
```

Comment:
do we need a fully qualified name here? can't we just specify this in our imports?